src/components/Calculator.js

```diff
@@ -1,89 +1,65 @@
-/* eslint jsx-quotes: ["error", "prefer-single"] */
 import React, { Component } from 'react';
+import Button from './Button';
+import calculate from '../logic/calculate';
+// import operate from '../logic/operate';
```

Comment:
- [Optional] Please, do not commit commented codes to your repo. Comments render your codebase messy.